shared/params.go

```diff
@@ -47,12 +48,15 @@ func (filter TestRunFilter) ToQuery(complete bool) (q url.Values) {
 			q.Add("product", p.String())
 		}
 	}
-	if filter.Complete || (complete && len(q) == 0) {
+	if filter.Complete || (completeIfDefault && len(q) == 0) {
```

Comment:
On a second thought, this logic introduced in #318 was incorrect. We probably need to differentiate not setting `complete` and setting `complete=false` by using `*bool` instead of `bool`.

E.g. https://test-run-filter-dot-wptdashboard-staging.appspot.com/results/?complete=false , I'd expect to see the latest run from each browser (i.e. not necessarily complete runs). Currently the `complete=false` is simply overridden here.

Reply:
Done.